### PR TITLE
task5737 controller role user

### PR DESCRIPTION
**Pull request by dangvancuong**
https://edu-redmine.framgia.vn/issues/5737

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use App\Role;
+use DB;
+use Session;
+use Hash;
+use Input;
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 13</br>There must be one blank line after the last USE statement; 0 found;

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+     */
+    public function store(CreateUserRequest $request)
+    {
+        if (!empty($request->password)) {
+            $password = trim($request->password);
+        } else {
+            # set the manual password
+            $length = 10;
+            $keyspace = '123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ';
+            $str = '';
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 56</br>Blank line found at start of control structure

File: app/Http/Controllers/UserController.php [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 92</br>String "manage.users.show" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $roles = Role::all();
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 106</br>String "manage.users.edit" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+            $user->password = Hash::make($str);
+        } elseif ($request->password_options == 'manual') {
+            $user->password = Hash::make($request->password);
+        }
+        $user->save();
+        $user->syncRoles($request->roles);
+
+        return redirect()->route('users.show', $id);
+        // if () {
+        //   return redirect()->route('users.show', $id);
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 140</br>Line exceeds 120 characters; contains 123 characters

File: app/Http/Controllers/UserController.php [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+     */
+    public function store(CreateUserRequest $request)
+    {
+        if (!empty($request->password)) {
+            $password = trim($request->password);
+        } else {
+            # set the manual password
+            $length = 10;
+            $keyspace = '123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ';
+            $str = '';
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 56</br>Blank line found at start of control structure

File: app/Http/Controllers/UserController.php [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 92</br>String "manage.users.show" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $roles = Role::all();
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 106</br>String "manage.users.edit" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+            $user->password = Hash::make($str);
+        } elseif ($request->password_options == 'manual') {
+            $user->password = Hash::make($request->password);
+        }
+        $user->save();
+        $user->syncRoles($request->roles);
+
+        return redirect()->route('users.show', $id);
+        // if () {
+        //   return redirect()->route('users.show', $id);
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 140</br>Line exceeds 120 characters; contains 123 characters

File: app/Http/Controllers/RoleController.php [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+        }
+        Session::flash('success', 'Successfully update the ' . $role->display_name . ' role in the database.');
+
+        return redirect()->route('roles.show', $id);
+    }
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 108</br>Opening brace should be on a new line

File: app/Http/Controllers/RoleController.php [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+     */
+    public function destroy($id) {
+        //
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 111</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/config/laraflash.php [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+        'priority' => 5,
+        /**
+         * Notification Title
+         *  (ie. "Success: ", "Warning: ", "Notice: ", null)
+         */
+        'title' => 'Notice: ',
+        /**
+         * Notification Content
+         *  This will probably never revert to this default,
+         *  but is set just in case
```
Comment (framgiaciv3) [inline app/Http/Controllers/config/laraflash.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laraflash.php</br>**Line**: 53</br>Closing bracket of array declaration must be on a new line

File: app/Http/Controllers/config/laratrust_seeder.php [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+        'contributor' => [
+            'profile' => 'r,u',
+        ],
+        'supporter' => [
+            'profile' => 'r,u',
+        ],
+        'subscriber' => [
+            'profile' => 'r,u',
+        ],
+    ],
```
Comment (framgiaciv3) [inline app/Http/Controllers/config/laratrust_seeder.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laratrust_seeder.php</br>**Line**: 31</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PermissionController.php [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+
+    public function store(CreatePermissionRequest $request)
+    {
+        $crud = $request->crud_selected;
+        foreach ($crud as $x) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/PermissionController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/PermissionController.php</br>**Line**: 47</br>String " " does not require double quotes; use single quotes instead

File: app/Http/Requests/CreateUserRequest.php [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
```
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 28</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateUserRequest.php [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+            'name' => 'required|max:255',
+            'email' => 'required|email|unique:users'
+        ];
+    }
+    public function messages()
+    {
+        return [
+            'name.max' => 'Số kí tự phải nhỏ hơn 255 ký tự',
+            'name.required' => 'Name không được để trống',
+            'email.required' => 'Email không được để trống',
```
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateRoleRequest.php [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function rules()
+    {
+        return [
+            'display_name' => 'required|max:255',
```
Comment (framgiaciv3) [inline app/Http/Requests/CreateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateRoleRequest.php</br>**Line**: 23</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/UpdateRoleRequest.php [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function rules()
+    {
+        $roleId = $this->route('role');
+
+        return [
+            'display_name' => 'required|max:255',
```
Comment (framgiaciv3) [inline app/Http/Requests/UpdateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateRoleRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/UserController.php [hunk 19]
```diff
@@ -0,0 +1,154 @@
...
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 91</br>String "manage.users.show" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 20]
```diff
@@ -0,0 +1,154 @@
...
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $roles = Role::all();
+        $user = User::where('id', $id)->with('roles')->first();
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 105</br>String "manage.users.edit" does not require double quotes; use single quotes instead

File: app/Http/Controllers/UserController.php [hunk 21]
```diff
@@ -0,0 +1,154 @@
...
+            $user->password = Hash::make($str);
+        } elseif ($request->password_options == 'manual') {
+            $user->password = Hash::make($request->password);
+        }
+        $user->save();
+        $user->syncRoles($request->roles);
+
+        return redirect()->route('users.show', $id);
+        // if () {
+        //   return redirect()->route('users.show', $id);
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 139</br>Line exceeds 120 characters; contains 123 characters

File: app/Http/Controllers/RoleController.php (hunk 10 above)
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 108</br>Opening brace should be on a new line

File: app/Http/Controllers/RoleController.php (hunk 11 above)
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 111</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/config/laraflash.php (hunk 12 above)
Comment (framgiaciv3) [inline app/Http/Controllers/config/laraflash.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laraflash.php</br>**Line**: 53</br>Closing bracket of array declaration must be on a new line

File: app/Http/Controllers/config/laratrust_seeder.php (hunk 13 above)
Comment (framgiaciv3) [inline app/Http/Controllers/config/laratrust_seeder.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laratrust_seeder.php</br>**Line**: 31</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PermissionController.php (hunk 14 above)
Comment (framgiaciv3) [inline app/Http/Controllers/PermissionController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/PermissionController.php</br>**Line**: 47</br>String " " does not require double quotes; use single quotes instead

File: app/Http/Requests/CreateUserRequest.php (hunk 15 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 28</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateUserRequest.php (hunk 16 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateRoleRequest.php (hunk 17 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateRoleRequest.php</br>**Line**: 23</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/UpdateRoleRequest.php (hunk 18 above)
Comment (framgiaciv3) [inline app/Http/Requests/UpdateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateRoleRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/UpdateUserRequest.php [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+        return true;
+    }
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function rules()
+    {
+        $userId = $this->route('user');
```
Comment (framgiaciv3) [inline app/Http/Requests/UpdateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.Incorrect</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateUserRequest.php</br>**Line**: 22</br>Language constructs must be followed by a single space; expected "return [" but found "return["

File: app/Http/Controllers/RoleController.php (hunk 10 above)
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 108</br>Opening brace should be on a new line

File: app/Http/Controllers/RoleController.php (hunk 11 above)
Comment (framgiaciv3) [inline app/Http/Controllers/RoleController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/RoleController.php</br>**Line**: 111</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/config/laraflash.php (hunk 12 above)
Comment (framgiaciv3) [inline app/Http/Controllers/config/laraflash.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laraflash.php</br>**Line**: 53</br>Closing bracket of array declaration must be on a new line

File: app/Http/Controllers/config/laratrust_seeder.php (hunk 13 above)
Comment (framgiaciv3) [inline app/Http/Controllers/config/laratrust_seeder.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laratrust_seeder.php</br>**Line**: 31</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/PermissionController.php (hunk 14 above)
Comment (framgiaciv3) [inline app/Http/Controllers/PermissionController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/PermissionController.php</br>**Line**: 47</br>String " " does not require double quotes; use single quotes instead

File: app/Http/Requests/CreateUserRequest.php (hunk 15 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 28</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateUserRequest.php (hunk 16 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateUserRequest.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/CreateRoleRequest.php (hunk 17 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateRoleRequest.php</br>**Line**: 23</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/UpdateRoleRequest.php (hunk 18 above)
Comment (framgiaciv3) [inline app/Http/Requests/UpdateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateRoleRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/UpdateUserRequest.php (hunk 22 above)
Comment (framgiaciv3) [inline app/Http/Requests/UpdateUserRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.Incorrect</br>**Severity**: error</br>**File**: app/Http/Requests/UpdateUserRequest.php</br>**Line**: 22</br>Language constructs must be followed by a single space; expected "return [" but found "return["

File: app/Http/Requests/CreateRoleRequest.php (hunk 17 above)
Comment (framgiaciv3) [inline app/Http/Requests/CreateRoleRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/CreateRoleRequest.php</br>**Line**: 23</br>Each line in an array declaration must end in a comma

**Comment by dangvancuong** [comment]
ready

**Pull request merged**